src/main/java/com/sandvoxel/generitech/common/tileentities/machines/TileEntityPulverizer.java

```diff
@@ -209,12 +207,29 @@ public long getPower() {
     @Override
     public void update() {
 
-        //container.givePower(1000,false);
-
-
+        //checking for the machine type
         if (machineTier == null)
             machineTier = MachineTier.byMeta(getBlockMetadata());
 
+
+        if (container.getStoredPower() <= 0 && machineTier != MachineTier.TIER_0 && machineActive != false || fuelRemaining == 0 && machineTier == MachineTier.TIER_0 && machineActive != false) {
+            machineActive = false;
+            this.markForUpdate();
+        }
+        if (fuelRemaining > 0 && machineTier == MachineTier.TIER_0 && machineActive == false) {
+            machineActive = true;
+            this.markForUpdate();
+        }
+        if (container.getStoredPower() > 0 && machineTier != MachineTier.TIER_0 && machineActive == false) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS1125)